app/models/server/models/Messages.js

```diff
@@ -107,14 +107,14 @@ export class Messages extends Base {
 		return this.createWithTypeRoomIdMessageAndUser('r', roomId, roomName, user, extraData);
```

Comment:
Auto Translate: No PR. Shall be PRed to RC

Reply:
#16363

Reply:
#16363